hapi/hedera-protobuf-java-api/src/main/proto/fees/fee_schedule.proto

```diff
@@ -94,6 +94,11 @@ enum Extra {
     ALLOWANCES=14;
     AIRDROPS=15;
     HOOKS=16;
```

Comment:
May be we can name this as `HOOK_UPDATES` , then we can name `CRYPTO_TRANSFER_WITH_HOOKS` as `HOOK_EXECUTION`